build.gradle

```diff
@@ -130,6 +130,16 @@ tasks.withType(FindBugs) {
   }
 }
 
+jacocoTestReport {
```

Comment:
By default the `jacoco` plugin does generate HTML. Is there a particular reason for making that plugin configuration explicit?